src/style.css

```
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
```

```suggestion
@font-face {
  font-family: Avenir, sans-serif;
  src: url(fonts/Avenir-Book.ttf);
  font-weight: 400;
}

@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir-Heavy.ttf);
  font-weight: 700;
}
```

No need to declare two different font-families.
 You can just call it bot 'Avenir' with different font-weights and then use it in your code like:

```
  font-family: Avenir, sans-serif;
  font-weight: 400;
```
and
```
  font-family: Avenir, sans-serif;
  font-weight: 700;
```
when needed